panther_hardware_interfaces/test/src/roboteq_mock.cpp

```diff
@@ -21,31 +21,48 @@ namespace panther_hardware_interfaces_test
 
 void RoboteqSlave::SetPosition(DriverChannel channel, std::int32_t value)
 {
-  (*this)[0x2106][static_cast<std::uint8_t>(channel)] = value;
+  (*this)[0x2104][static_cast<std::uint8_t>(channel)] = value;
 }
 
-void RoboteqSlave::SetVelocity(DriverChannel channel, std::int32_t value)
+void RoboteqSlave::SetVelocity(DriverChannel channel, std::int16_t value)
 {
-  (*this)[0x2106][static_cast<std::uint8_t>(channel) + 2] = value;
+  (*this)[0x2107][static_cast<std::uint8_t>(channel)] = value;
 }
 
-void RoboteqSlave::SetCurrent(DriverChannel channel, std::int32_t value)
+void RoboteqSlave::SetCurrent(DriverChannel channel, std::int16_t value)
 {
-  (*this)[0x2106][static_cast<std::uint8_t>(channel) + 4] = value;
+  (*this)[0x2100][static_cast<std::uint8_t>(channel)] = value;
 }
```

Comment:
Perhaps move it to the .hpp file, following the pattern of other methods with `static_cast`?